Controller/CRUDController.php

```diff
@@ -63,6 +63,9 @@ public function setContainer(ContainerInterface $container = null)
      */
     public function render($view, array $parameters = array(), Response $response = null)
     {
+        if (!$this->isXmlHttpRequest()) {
+            $parameters['breadcrumbs_builder'] = $this->get('sonata.admin.breadcrumbs_builder');
+        }
```

Comment:
Alternatives : 

- use a global twig variable (eeew, this is bad)
- use an embed controller to render the menu and the head (would add 2 sub-request, so not optimal)

What do you think @sonata-project/contributors ?